checker/src/main/java/org/checkerframework/checker/toy/ToyChecker.java

```diff
@@ -0,0 +1,6 @@
+package org.checkerframework.checker.toy;
```

Comment:
Defining this in the main Checker Framework means that it will appear in distributions and be visible to users.
I suggest that this be defined under checker/tests.

Reply:
https://github.com/typetools/checker-framework/blob/master/framework/src/test/java/lubglb/LubGlbChecker.java is a similar test checker and something relative to it would work well.